packages/firebase_admob/README.md

```diff
@@ -73,6 +73,8 @@ Starting in version 7.42.0, you are required to add your AdMob app ID in your **
 
 Failure to add this tag will result in the app crashing at app launch with a message including *"GADVerifyApplicationID."*
 
+You are also required to create an "App" in firebase and generate a GoogleService-info.plist file. This file needs to be embedded in the projects "Runner/Runner" folder using Xcode. If this step isn't done, the app will crash on startup.
```

Comment:
Since this is required for Android and iOS, I think it would be better to create a section for "Configuring a Firebase Project". It would also be beneficial to point to documentation from Firebase about how to create an "App". Some documentation I found for Android:
https://firebase.google.com/docs/android/setup#create-firebase-project  -> Steps 1-3.1
for iOS:
https://firebase.google.com/docs/ios/setup#create-firebase-project -> Steps 1-3

Reply:
Good point. I will add a section for each.